zpages/templates.go

```diff
@@ -116,3 +114,83 @@ func traceIDFormatter(r traceRow) template.HTML {
 func even(x int) bool {
 	return x%2 == 0
 }
+
+func latency(ws *windowStat) string {
+	_, _, diff := ws.read()
+	if diff == nil {
+		return "-"
+	}
+	return msFormatter(diff.Mean)
+}
+
+func totalLatency(ws *windowStat) string {
+	if ws.lastUpdate == -1 {
+		return "-"
+	}
+	return msFormatter(ws.intervals[ws.lastUpdate].distribution.Mean)
+}
+
+func count(ws *windowStat) string {
+	_, _, diff := ws.read()
+	if diff == nil {
+		return countFormatter(0)
+	}
+	return countFormatter(diff.Count)
+}
+
+func totalCount(ws *windowStat) string {
+	if ws.lastUpdate == -1 {
+		return countFormatter(0)
+	}
+	return countFormatter(ws.intervals[ws.lastUpdate].distribution.Count)
+}
+
+func dataRate(ws *windowStat) string {
+	start, end, diff := ws.read()
+	if diff == nil {
+		return dataRateFormatter(0)
+	}
+	seconds := float64(end.Sub(start) / time.Second)
```

Comment:
I'd be more comfortable using the method `Seconds` e.g.
```go
seconds := end.Sub(start).Seconds()
```
makes for more clarity

Reply:
I am computing the seconds between start and end. The Seconds() function does not convert to seconds, it just gives you the seconds component of the time.